lib/powerbar.rb

```diff
@@ -308,14 +308,20 @@ def h_done
   end
 
   def terminal_width
+    require 'io/console'
+    rows, cols = IO.console.winsize
+    cols - 1
+  rescue LoadError #io/console not in ruby < 1.9.3
```

Comment:
I think we should just drop support for Ruby <1.9.3 and for Solaris at this point.

If someone needs this gem on these old platforms they can just stick to an older gem version.

Reply:
That simplifies the method a bit. I'll push a commit that has terminal width using just io/console.

Reply:
Actually, it doesn't seem to be necessary on OS X either, at least with up-to-date versions of Terminal.app and iTerm2.app